src/Components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,70 @@
+import "./Sidebar.css";
+import Item from "../Item/Item";
+import InfoIcon from "@mui/icons-material/Info";
+import LanguageIcon from "@mui/icons-material/Language";
+import LocationOnIcon from "@mui/icons-material/LocationOn";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import ArrowBackIosNewIcon from "@mui/icons-material/ArrowBackIosNew";
+import { useState } from "react";
+
+const Sidebar = () => {
+  const [isCollapsed, setisCollapsed] = useState(false);
+  const items = [
```

Comment:
Place items in a seperate file such as constants or enum. As there is a possibility that we may use it somewhere else later on.